lib/MusicBrainz/Server/Controller/Admin.pm

```diff
@@ -159,6 +159,43 @@ sub edit_banner : Path('/admin/banner/edit') Args(0) RequireAuth(banner_editor)
     }
 }
 
+sub email_search : Path('/admin/email-search') Args(0) RequireAuth(account_admin) {
+    my ($self, $c) = @_;
+
+    my $form = $c->form(form => 'Admin::EmailSearch');
+    my @results;
+
+    if ($c->form_submitted_and_valid($form)) {
+        @results = $c->model('Editor')->search_by_email(
+            $form->field('email')->value // '',
```

Comment:
Even though it is accessible to admin only, it should probably be checked for potential SQL injection.

Reply:
Hmm, can you give an example? It's passed in via a normal DBI placeholder so should be fully escaped by the driver.

Reply:
Did not find Little Bobby Tables: https://metacpan.org/pod/DBI#Placeholders-and-Bind-Values